qmk_cli/script_qmk.py

```diff
@@ -82,7 +82,10 @@ def main():
     if qmk_firmware.exists():
         os.chdir(str(qmk_firmware))
         sys.path.append(str(qmk_firmware / 'lib' / 'python'))
-        import qmk.cli
+        try:
+            import qmk.cli
+        except ImportError:
+            pass # Ingore as this might folder might not be set up correctly
```

Comment:
```suggestion
            print('Error: qmk_firmware at %s is either too old or not set up correctly!.')
            print('Please either update it or remove it completely before continuing.')
```

Reply:
In the case of an empty folder, this will still print out. The user would see the following
```console
$ qmk setup zvecr/qmk_firmware
Error: qmk_firmware at %s is either too old or not set up correctly!.
Please either update it or remove it completely before continuing.
☒ qmk_firmware not found!
*** Would you like to clone zvecr/qmk_firmware? [y/n]
```

I think parts of the message apply but i'm not sold on the overall output in this context.

Reply:
Oops, forgot about the substitution. Check now.